webpack/styles/widgets/_recentitems.scss

```diff
@@ -34,20 +34,24 @@
 
 .recent-item-list {
   @extend .list-group-item;
-  padding-right: 15px;
-  padding-left: 15px;
-  padding-top: 18px;
-  padding-bottom: 36px;
-  border: 0px;
-  font-weight: bold;
-  color: $cdc-blue;
-  font-size: 16px;
+  padding-right: 15px !important;
+  padding-left: 15px !important;
+  margin-bottom: 5px;
+  margin-top: 5px;
+  border: 0px !important;
+  font-weight: bold !important;
```

Comment:
This might not be the correct way to do this but I added important because changing the analytics widgets to buttons caused them to get button formatting without the !important tag. If there is a better approach please let me know.